### DISCUSSION TITLE
Return more than one collection of different classes with Index

### DISCUSSION THREAD

**Discussion by wrafanunes** [ravendb/ravendb#14131] [General]
Hello guys, I have four kinds of reports, and I want a way to get all the report collections, but using just one index, is there a way to make an index get different collections from differente classes from my database?

**Comment by krivulcik** [comment]
You can use a multi-map index for that.

See https://ravendb.net/docs/article-page/5.3/csharp/studio/database/indexes/create-multi-map-index for more details.

**Reply by krivulcik** [reply]
Possibly more useful documentation: https://ravendb.net/docs/article-page/5.3/csharp/indexes/multi-map-indexes

**Reply by wrafanunes** [reply]
Thanks man, it worked just fine
